package.json

```diff
@@ -2,13 +2,14 @@
   "name": "spectacle-boilerplate",
   "version": "1.0.1",
   "description": "ReactJS Powered Presentation Framework",
-  "main": "lib/index.js",
+  "main": "dist/bundle.js",
   "scripts": {
-    "clean": "rimraf dist",
+    "prebuild": "npm run clean",
     "build": "cross-env NODE_ENV=production webpack --config webpack.config.production.js",
+    "clean": "rimraf dist",
     "lint": "eslint --ext .js,.jsx .",
-    "deploy": "npm run clean & npm run build && surge -p .",
-    "start": "cross-env NODE_ENV=development node server.js"
+    "deploy": "npm run clean & npm run build && surge -p dist",
+    "start": "cross-env NODE_ENV=development webpack-dev-server --hot --port 3000 --content-base dist --config webpack.config.development.js"
```

Comment:
Self-explanatory, but swap out the custom express server and just use webpack dev server. Hot reloading and everything (to the best of my knowledge/testing) still works!